content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+
+### Validation
+
+The automatically-provisioned [AWS overview dashboard][11] provides an immediate window into the metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="AWS overview dashboard">}}
+
+
+## Sending Logs
+
+As a result of setting up the Datadog Forwarder function with the CloudFormation template, log collection can be configured from directly within the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder lambda. With the ARN entered, click the check mark to establish the connection: 
+{{< img src="getting_started/integrations/integration-tile-collect-logs-1.png" alt="Datadog AWS integration tile 'Collect Logs' tab">}}
+
+Once completed, you'll be able to select from any of the log sources mentioned in [this guide][12] for sending AWS service logs. For this guide, we'll select only S3 access logs, and click `Save`:
+{{< img src="getting_started/integrations/integration-tile-collect-logs-2.png" alt="Datadog AWS integration tile 'Collect Logs' tab with 'S3' option checked">}}
+
+
+### Validation
+
+Find your logs in the [Logs Explorer][13] using either the `source` or `service` facets from the facet panel:
+{{< img src="getting_started/integrations/logs-explorer.png" alt="Datadog Logs Explorer">}}
+
+## Deeper visibility with the Datadog agent
+
+By default the Datadog AWS integration crawls the CloudWatch API for AWS-provided metrics, but you can gain even deeper visibility into your EC2 instances and ECS clusters with the [Datadog agent][14]. The agent is a lightweight daemon which reports metrics and events, and can also be configured for logs and traces. The [Agent Installation][15] section of the Datadog application provides instructions for installing the agent on a wide variety of operating systems, with many options--such as Amazon Linux 2--offering one-step installation commands. This command can then be run from the instance terminal to install the agent:
+{{< img src="getting_started/integrations/integrations-agent-installation.png" alt="Datadog agent installation page">}}
+
+
+Once the agent is installed, it's graphically represented within the [Infrastructure List][16] with a bone icon:
+{{< img src="getting_started/integrations/infrastructure-list.png" alt="Datadog infrastructure list">}}
+
+
+As can be seen in the screenshot above, the host with the Datadog agent is reporting data from the [System][17] and [NTP][18] checks. The System check provides metrics around CPU, memory, filesystem, and I/O, providing additional insights into the host. You can enable additional [integrations][19] to suit the environment and use case, and [DogStatsD][20] can additionally be leveraged for sending custom metrics directly to Datadog. 
+
+## What's next?
```

Comment:
I think this section is a great start but could and should be expanded a lot more. I would love to see info on ECS, EC2, EKS, Fargate, etc - I would classify it as the most popular of this list - https://docs.datadoghq.com/integrations/#cat-aws (and maybe even consider including a link to this list). I would like to see this section become a really powerful map to navigating through AWS in  Datadog.